### DISCUSSION TITLE
Suggestion (minor): allow user to change preferences even without any book loaded

### DISCUSSION THREAD

**Discussion by gladladvlad** [johnfactotum/foliate#1375] [Ideas]
Hello, hello. I have a very minor suggestion (inspired from real historical events). I'll try to be succint. Feel free to ignore if you don't think it's valid.

Here's my user "scenario": 
1. fresh install foliate,
2. run foliate,
3. first thing I notice: light mode,
4. (no book loaded yet) look for a preferences menu with intention to set dark mode.

Currently: user would not see the preferences button. Reason being: no book was opened/loaded.
Expected (by me, at least): user *would* see a preferences/settings button if one existed.

Now, I was playing around with the app as a first time user for a bit. I did load a book and restart and try out a bunch of things. But somehow, I managed to miss trying to look for the preferences while I had a book loaded. This might be entirely on me but I think it'd be helpful to allow the user to see the preferences menu to avoid potential wild goose chases. Maybe at least a message telling the user to load a book for the preferences menu?

Again, use your own judgement. I already figured it out so I'm good now lol.

**Comment by johnfactotum** [comment]
I guess this is #1250

**Reply by gladladvlad** [reply]
Ah, yes, you're right. I did do a search before posting but I guess I used different words and missed it.

I'm fine closing this and leaving only that one open.
